### PR TITLE
Ubuntu 20.04 Quick Guide improvements

### PR DESCRIPTION
**Pull request by enbrnz**
- Additional module php-opcache explicitly installed as required for supported setup
- Removed unnecessary packages
- Split out apt installation and pecl installation into 2 code boxes, as copy paste doesn't work otherwise
- Added additonal packages ldap-utils and smbclient for troubleshooting
- Cleaned up Apache configuration

### CODE REVIEW

File: modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc [hunk 1]
```diff
@@ -61,14 +61,23 @@ apt install -y \
...
   php-json php-mbstring php-mysql \
   php-ssh2 php-xml php-zip \
-  php-apcu php-redis php-ldap 
+  php-apcu php-redis php-ldap \
+  php-opcache
 ----
 
 === Install smbclient php module
 
+First run the installation of the required packages
```
Comment (EParzefall) [inline modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc:70]:
```suggestion
First install the required packages:
```

File: modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc [hunk 2]
```diff
@@ -81,38 +90,42 @@ Check if it was successfully activated
...
-  imagemagick unzip
+libsmbclient
+smbclient
 ----
 
-=== Configure Apache
+=== Install the Recommended Packages
 
-==== Change the Document Root
+Additional useful tools helpful for debugging
```
Comment (EParzefall) [inline modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc:103]:
```suggestion
Additional useful tools helpful for debugging:
```

**Review by EParzefall — APPROVED** [review]
LGTM

**Comment by EParzefall** [comment]
Backports to 10.9 and 10.8, correct?

**Comment by enbrnz** [comment]
Thanks, for your help @EParzefall

**Pull request merged**